common/lib/capa/capa/templates/chemicalequationinput.html

```diff
@@ -2,23 +2,25 @@
 <div id="chemicalequationinput_${id}" class="chemicalequationinput">
     <div class="script_placeholder" data-src="${previewer}"/>
 
-    <div class="${status.classname}" id="status_${id}">
+    <div class="status ${status.classname}">
```

Comment:
Why did you put a class of "status" here? With the actual status spans included within (line 16), I'm wondering why the wrapper for the input itself also has this status class.